tests/Bootstrap.php

```diff
@@ -16,9 +16,11 @@
  * and is licensed under the MIT license.
  */
 
-ini_set('error_reporting', E_ALL);
+declare(strict_types=1);
```

Comment:
File can probably be dropped overall

Reply:
I added the test namespaces to autoload-dev to get rid of this file.